package error_handling_with_exceptions.task1;public class Task1 {
}
